sunpy/net/tests/test_scraper.py

```diff
@@ -225,3 +224,31 @@ def test_yearly_overlap():
     # Should return a single file for 2013
     trange = TimeRange("2013-01-02", "2013-01-03")
     assert len(scraper.filelist(trange)) == 1
+
+
+def test_scraper_http_error(endpoint=404):
```

Comment:
Why is the number an input to the function?

Reply:
You should read on how pytest can use parameters to do this. It should not be hard coded to the function input.

Reply:
But we should make sure we test all the error codes.